verticapy/core/string_sql/base.py

```diff
@@ -100,7 +100,7 @@ def _in(self, *args) -> "StringSQL":
         elif len(args) == 0:
             ValueError("Method 'in_' doesn't work with no parameters.")
         else:
-            x = [elem for elem in args]
+            x = copy.deepcopy(args)
```

Comment:
in a previous place, you used list() to replace the [elem ...]

Reply:
it depends on the use-case. List will also convert dict to list.